Cargo.toml

```diff
@@ -1,58 +1,79 @@
 [package]
-authors = ["Without Boats <lee@libertad.ucsd.edu>", "Pascal Hertleif <killercup@gmail.com>", "Sebastian Garrido <sebasgarcep@gmail.com>", "Jonas Platte <mail@jonasplatte.de>", "Benjamin Gill <git@bgill.eu>", "Andronik Ordian <write@reusable.software>"]
-categories = ["development-tools", "development-tools::cargo-plugins"]
-description = "This extends Cargo to allow you to add and remove dependencies by modifying your `Cargo.toml` file from the command line. It contains `cargo add`, `cargo rm`, and `cargo upgrade`."
-documentation = "http://killercup.github.io/cargo-edit/"
-homepage = "https://github.com/killercup/cargo-edit"
-keywords = ["cargo", "cargo-subcommand", "cli", "dependencies", "crates"]
-license = "Apache-2.0/MIT"
-name = "cargo-edit"
-readme = "README.md"
-repository = "https://github.com/killercup/cargo-edit"
-version = "0.2.0"
+authors = [
+    'Without Boats <lee@libertad.ucsd.edu>',
+    'Pascal Hertleif <killercup@gmail.com>',
+    'Sebastian Garrido <sebasgarcep@gmail.com>',
+    'Jonas Platte <mail@jonasplatte.de>',
+    'Benjamin Gill <git@bgill.eu>',
+    'Andronik Ordian <write@reusable.software>',
+]
+categories = [
+    'development-tools',
+    'development-tools::cargo-plugins',
+]
+description = 'This extends Cargo to allow you to add and remove dependencies by modifying your `Cargo.toml` file from the command line. It contains `cargo add`, `cargo rm`, and `cargo upgrade`.'
+documentation = 'http://killercup.github.io/cargo-edit/'
+homepage = 'https://github.com/killercup/cargo-edit'
+keywords = [
+    'cargo',
+    'cargo-subcommand',
+    'cli',
+    'dependencies',
+    'crates',
+]
+license = 'Apache-2.0/MIT'
+name = 'cargo-edit'
+readme = 'README.md'
+repository = 'https://github.com/killercup/cargo-edit'
+version = '0.2.0'
 [[bin]]
```

Comment:
This format is pretty (😄), except for the missing newlines around these sections